SpatialGDK/Source/SpatialGDKEditor/SpatialGDKEditor.Build.cs

```diff
@@ -30,6 +30,7 @@ public SpatialGDKEditor(ReadOnlyTargetRules Target) : base(Target)
 				"Slate",
 				"SlateCore",
 				"SpatialGDK",
+				"SpatialGDKFunctionalTests",
```

Comment:
This is probably fine, I think, but @joshuahuburn is the module dependency police.